conans/test/functional/cache/path_limit_test.py

```diff
@@ -111,7 +111,7 @@ def upload_test(self):
             files = os.listdir(package_folder)
             self.assertIn("myfile.txt", files)
             self.assertIn("myfile2.txt", files)
-            self.assertNotIn("conan_package.tgz", files)
+            self.assertIn("conan_package.tgz", files)
```

Comment:
I think this is concerning, to have the .tgz there, that might also be deployed with the ``-g deploy`` or ``deploy()`` methods.